odbc_loader_package/Makefile

```diff
@@ -22,8 +22,17 @@ test:
 	## Try uninstalling first, just in case we have a stale version around
 	-@$(MAKE) uninstall
 	@$(MAKE) install
-	-diff -u Tests.out <($(VSQL) -f examples/sample_usage.sql 2>&1)
-	@$(MAKE) uninstall
+	@trap '$(MAKE) uninstall' EXIT; $(MAKE) test_impl
+
```

Comment:
It might be helpful to indent the first perl expression the same way the second one is.

Is there a reason the "Error parsing" clause is missing from the first perl expression?

Reply:
Uhh - no need to run diff/perl twice.  I'll fix that.